.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: node:6.10.0
```

Comment:
Since it is a library we should take advantage of circleci and run the tests against multiple node versions.
You can define multiple jobs, all similar except the node version in the docker image, and in the workflow run all these jobs. I think we should at least run with the lowest supported version and the current one

Reply:
ok, added all major versions plus the minimum required.